chia/full_node/mempool.py

```diff
@@ -24,26 +23,58 @@ class MempoolRemoveReason(Enum):
 
 class Mempool:
     def __init__(self, mempool_info: MempoolInfo, fee_estimator: FeeEstimatorInterface):
-        self.log: logging.Logger = logging.getLogger(__name__)
-        self.spends: Dict[bytes32, MempoolItem] = {}
-        self.sorted_spends: SortedDict = SortedDict()
+        self._spends: Dict[bytes32, MempoolItem] = {}
+        self._sorted_spends: SortedDict = SortedDict()
         self.mempool_info: MempoolInfo = mempool_info
         self.fee_estimator: FeeEstimatorInterface = fee_estimator
-        self.removal_coin_id_to_spendbundle_ids: Dict[bytes32, List[bytes32]] = {}
-        self.total_mempool_cost: CLVMCost = CLVMCost(uint64(0))
-        self.total_mempool_fees: int = 0
+        self._removal_coin_id_to_spendbundle_ids: Dict[bytes32, List[bytes32]] = {}
+        self._total_mempool_cost: CLVMCost = CLVMCost(uint64(0))
+        self._total_mempool_fees: int = 0
+
+    def total_mempool_fees(self) -> int:
+        return self._total_mempool_fees
+
+    def total_mempool_cost(self) -> CLVMCost:
+        return self._total_mempool_cost
+
+    def all_spends(self) -> List[MempoolItem]:
+        return list(self._spends.values())
+
+    def all_spend_ids(self) -> List[bytes32]:
+        return list(self._spends.keys())
+
+    def spends_by_feerate(self) -> List[MempoolItem]:
+        ret: List[MempoolItem] = []
+        for spends_with_fpc in reversed(self._sorted_spends.values()):
+            ret.extend(spends_with_fpc.values())
+        return ret
```

Comment:
For these and the ones below, I'd vote for taking this opportunity to properly name these with "mempool items" instead of "spends".

Reply:
I'm not so invested in these variable names as I already have a patch that replaces it all (locally)

Reply:
oh, you mean the member function should be `mempool_items_by_feerate()`?

Reply:
Yep, interface level proper naming.

Reply:
`mempool_item` is a bit long. is `tx` too short?

Reply:
Sure, sounds like a plan.